pipelines/incubator/validate-active-stack-task.yaml

```diff
@@ -139,15 +147,15 @@ spec:
               fi    
        
               if [ "$TARGET_DIGEST" =  "$CURRENT_DIGEST" ]; then
-                 echo "The Kabanero stack contained in $STACK is active on this system and can be built."
+                 echo "The Kabanero stack, $STACK, in $APPSODY_CONFIG is active on this cluster and can be built."
                  exit 0
               else
                  echo "Version: $STACK_VERSION, Digest: $CURRENT_DIGEST, Target Digest: $TARGET_DIGEST"
               fi
            done
         
-           echo "The Kabanero stack contained in "$STACK" is not active on this system and contained in the image registry and cannot be built."
-           exit 1
+           echo "WARNING: A compatible version of the Kabanero stack, "$STACK", is not active on this cluster. Please review the active versions of the stack on the cluster and the stack specification in the $APPSODY_CONFIG file of the git project."
```

Comment:
/of the Kabanero stack/of the application stack/

should you add "oc get stacks -n <kabanero-namespace>" to the message for how to check?

Reply:
Thanks for the feedback.  Updated but used `(oc get stack $STACK_NAME -o json)` instead of `oc get stacks` as that doesn't directly say the activated version(s).